tests/test_e2e_05_topology.py

```diff
@@ -229,3 +231,150 @@ def test_090_disabling_link_persistent(self):
 #        # /api/kytos/topology/v3/switches --> check if it is enabled
 #        # kill kytosd and restart, check if switches are still enabled
 #        # check topology discovery
+
+    def test_disabled_switch_on_start(self):
+
+        sw = "00:00:00:00:00:00:00:01"
+
+        # Start the controller setting an environment in
+        # which all elements are disabled in a clean setting
+        self.net.start_controller(clean_config=True, enable_all=False)
+        self.net.wait_switches_connect()
+
+        # Make sure the switch is disabled
+        api_url = KYTOS_API + '/topology/v3/switches'
+        response = requests.get(api_url)
+        data = response.json()
+
+        assert response.status_code == 200
+        assert data['switches'][sw]['enabled'] is False
+
+    def test_disabling_switch(self):
+
+        sw = "00:00:00:00:00:00:00:01"
+
+        # Start the controller setting an environment in
+        # which all elements are disabled in a clean setting
+        self.net.start_controller(clean_config=True, enable_all=False)
+        self.net.wait_switches_connect()
+
+        # Enable the switch
+        api_url = KYTOS_API + '/topology/v3/switches/%s/enable' % sw
+        response = requests.post(api_url)
+        assert response.status_code == 201
+
+        # Start the controller setting an environment in which the setting is
+        # preserved (persistence) and avoid the default enabling of all elements
+        self.net.start_controller(clean_config=False, enable_all=False)
+        self.net.wait_switches_connect()
+
+        # Check if the switch is enabled
+        api_url = KYTOS_API + '/topology/v3/switches'
+        response = requests.get(api_url)
+        data = response.json()
+
+        assert response.status_code == 200
+        assert data['switches'][sw]['enabled'] is True
+
+        # Disable the switch and check if the switch is disabled
```

Comment:
Up to this point the test was for enabling a switch, and now you are testing to disable the switch. Maybe you should rename the test or create two separed